app/experimenter/experiments/tests/test_views.py

```diff
@@ -177,6 +177,84 @@ def test_filters_by_firefox_channel(self):
             set(Experiment.objects.filter(firefox_channel=include_channel)),
         )
 
+    def test_list_filters_by_search_text(self):
+        user_email = "user@example.com"
+
+        exp_1 = ExperimentFactory.create_with_status(
+            random.choice(Experiment.STATUS_CHOICES)[0],
+            name="Experiment One Cat",
+            short_description="",
+            slug="exp-1",
+            related_work="",
+            addon_experiment_id="1",
+            pref_key="",
+            public_name="",
+            public_description="",
+            objectives="",
+            analysis="",
+            analysis_owner="",
+            engineering_owner="",
+            bugzilla_id="4",
+            normandy_slug="",
+        )
+
+        exp_2 = ExperimentFactory.create_with_status(
+            random.choice(Experiment.STATUS_CHOICES)[0],
+            name="Experiment Two Cat",
+            short_description="",
+            slug="exp-2",
+            related_work="",
+            addon_experiment_id="2",
+            pref_key="",
+            public_name="",
+            public_description="",
+            objectives="",
+            analysis="",
+            analysis_owner="",
+            engineering_owner="",
+            bugzilla_id="5",
+            normandy_slug="",
+        )
+
+        exp_3 = ExperimentFactory.create_with_status(
+            random.choice(Experiment.STATUS_CHOICES)[0],
+            name="Experiment Three Dog",
+            short_description="",
+            slug="exp-3",
+            related_work="",
+            addon_experiment_id="3",
+            pref_key="",
+            public_name="",
+            public_description="",
+            objectives="",
+            analysis="",
+            analysis_owner="",
+            engineering_owner="",
+            bugzilla_id="6",
+            normandy_slug="",
+        )
+
```

Comment:
So, I had to override all the pertinent fields that Factory Boy creates because it was actually occasionally generating the words "Dog" and "Cat" in some of the fields.. which would then make the tests fail.  😆 If there was a uniqueness constraint issue I put in something... unique.